rdrf/rdrf/forms/registration_forms.py

```diff
@@ -0,0 +1,122 @@
+from operator import attrgetter
+import pycountry
+from django.forms import CharField, ChoiceField, DateField, BooleanField
+from django.forms.widgets import RadioSelect, Select
+from django.utils.translation import gettext as _
+
+from registration.forms import RegistrationForm
+from rdrf.helpers.utils import get_preferred_languages
+
+
+def _tuple(code, name):
+    return code, _(name)
+
+
+def _countries():
+    countries = sorted(pycountry.countries, key=attrgetter('name'))
+    result = [_tuple("-1", "Country")]
+    return result + [_tuple(c.alpha_2, c.name) for c in countries]
+
+
+def _preferred_languages():
+    languages = get_preferred_languages()
+    return [_tuple(l.code, l.name) for l in languages] if languages else [_tuple('en', 'English')]
+
+
+class PatientRegistrationForm(RegistrationForm):
+
+    placeholders = {
+        'username': _("Username"),
+        'password1': _("Password"),
+        'password2': _("Repeat Password"),
+        'first_name': _("Given Names"),
+        'surname': _("Surname"),
+        'date_of_birth': _("Date of Birth"),
+        'address': _("Address"),
+        'suburb': _("Suburb / Town"),
+        'state': _("State / County / Province / Region"),
+        'postcode': _("Zip / Postal Code"),
+        'phone_number': _('Phone Number')
+    }
+
+    gender_choices = [
```

Comment:
Could this come from the models so we don't duplicate it here?

Reply:
Updated.